src/Nri/Ui/Select/V3.elm

```diff
@@ -0,0 +1,106 @@
+module Nri.Ui.Select.V3
+    exposing
+        ( Config
+        , customView
+        , view
+        )
+
+{-| Build a select input.
+
+
+## Upgrading to V3
+
+  - Remove the old styles. V3 uses compile-at-runtime elm-css!
+
+
+# Configure
+
+@docs Config
+
+
+# Render
+
+@docs view, customView
+
+-}
+
+import Css
+import Css.Foreign
+import Dict
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attributes exposing (..)
+import Html.Styled.Events exposing (..)
+import Json.Decode exposing (Decoder, andThen, succeed)
+import Nri.Ui
+import Nri.Ui.Colors.V1
+import Nri.Ui.Styles.V1
+import Nri.Ui.Util
+
+
+{-| Select-specific Choice.Config
+-}
+type alias Config a =
+    { choices : List { label : String, value : a }
+    , current : a
+    }
+
+
+{-| TODO: move this to View.Extra
```

Comment:
Is this todo still a thing we want to do?

Reply:
Huh, I guess to me this seems like a sort of strange thing to add to `View.Extra` since it is pretty nri-specific. How does moving this to `Nri.Ui.Util` sound?

Reply:
Or we can just not do it 🤷‍♀️ 😄

Reply:
Oh, and it has a `toString` function in it which is going 👋 ...

Reply:
Hah, I am going to just update the To Do...